### DISCUSSION TITLE
Meeting Guide Download/Printable Schedule

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/pdf#64] [General]
Hi - We downloaded the meeting guide to make a schedule and noticed that there is now space after every line where the previous months it wasn't. This causes the printable meeting guide to go from 60 pages to 90 which is a definite increase in costs. Is there anyway to change it back to no space instead of single spaced?
[Feb Download.pdf](https://github.com/code4recovery/12-step-meeting-list/files/14182892/Feb.Download.pdf)
[Meeting Guide 12.7.23 (1).pdf](https://github.com/code4recovery/12-step-meeting-list/files/14182899/Meeting.Guide.12.7.23.1.pdf)

**Comment by anchovie91471** [comment]
@ATLCOTech, thanks for your input!

I will see if I can tighten up the line spacing a little more.

I am going to move this discussion to the PDF service discussions since it's not specifically a 12 Step Meeting List issue.

**Reply by ATLCOTech** [reply]
Thanks Anthony! Sorry about that. I'm still trying to figure out using this platform. Thread chats are a bit intimidating but I know tech folks like it. Reddit makes my head hurt. LOL

**Reply by anchovie91471** [reply]
No worries! I completely understand!

I'll let you know when I make the changes to PDF.

**Comment by Jodyaasanjose** [comment]
Hi !  We are created a new meeting guide and it went from 40 pages to 52.  It looks like there is a space between line now. I see Atlanta may have had the same issue, but it looks like it is fixed for them now.  Can you guide me to fixing that please?

jody@aasanjose.org or my cell 925-250-0422

**Reply by anchovie91471** [reply]
It isn't fixed yet. I'll update this discussion when I get it fixed.

**Reply by Jodyaasanjose** [reply]
Thank you!  Do you have an ETA? Just wondering if I should go ahead and print a new guide now or if we can hold off until the change is made.

**Comment by anchovie91471** [comment]
Today or tomorrow I’d say.

**Reply by Jodyaasanjose** [reply]
You are awesome! Thanks

**Comment by anchovie91471** [comment]
@ATLCOTech @Jodyaasanjose 

Please check PDF with your feed now. I've tightened up the line spacing some. It should be fewer pages now. 

We are now supporting more languages with PDF, so the font we were using had to be changed to one that supports more characters. Because of the switch, the layout won't match 100%, but we are hoping to get it as close as possible.

Thanks for your patience and your feedback!
Anthony

**Reply by joshreisner** [reply]
hi @Jodyaasanjose you'd mentioned that the font was much smaller, is that still true or is it ok now?

**Comment by Jodyaasanjose** [comment]
Yes it went from 52 pages to 46.  Before the initial changes it was only 42

On Fri, Feb 9, 2024 at 8:02 AM Anthony Baggett ***@***.***>
wrote:

> @ATLCOTech <https://github.com/ATLCOTech> @Jodyaasanjose
> <https://github.com/Jodyaasanjose>
>
> Please check PDF with your feed now. I've tightened up the line spacing
> some. It should be fewer pages now.
>
> We are now supporting more languages with PDF, so the font we were using
> had to be changed to one that supports more characters. Because of the
> switch, the layout won't match 100%, but we are hoping to get it as close
> as possible.
>
> Thanks for your patience and your feedback!
> Anthony
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8420705>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BF6R3N37RGZ6VWZFXYRSFYTYSZCANAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRQG4YDK>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
In Grateful Service,

MLC

**Comment by Jodyaasanjose** [comment]
Yes it went from 52 pages to 46.  Before the initial changes it was only 42

On Fri, Feb 9, 2024 at 9:41 AM Josh Reisner ***@***.***>
wrote:

> hi @Jodyaasanjose <https://github.com/Jodyaasanjose> you'd mentioned that
> the font was much smaller, is that still true or is it ok now?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8421636>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BF6R3NYJTJE5XNNGUDBN3DDYSZNVTAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRRGYZTM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
In Grateful Service,

MLC

**Comment by anchovie91471** [comment]
Do you have a copy of the old 42 page one? Maybe if I see it in context, I
can make more adjustments.

On Fri, Feb 9, 2024 at 1:23 PM Jodyaasanjose ***@***.***>
wrote:

> Yes it went from 52 pages to 46. Before the initial changes it was only 42
>
> On Fri, Feb 9, 2024 at 8:02 AM Anthony Baggett ***@***.***>
> wrote:
>
> > @ATLCOTech <https://github.com/ATLCOTech> @Jodyaasanjose
> > <https://github.com/Jodyaasanjose>
> >
> > Please check PDF with your feed now. I've tightened up the line spacing
> > some. It should be fewer pages now.
> >
> > We are now supporting more languages with PDF, so the font we were using
> > had to be changed to one that supports more characters. Because of the
> > switch, the layout won't match 100%, but we are hoping to get it as
> close
> > as possible.
> >
> > Thanks for your patience and your feedback!
> > Anthony
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8420705>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BF6R3N37RGZ6VWZFXYRSFYTYSZCANAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRQG4YDK>
>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
> In Grateful Service,
>
> MLC
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8422538>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACF42JIWDY52OAY3AJLWEWDYSZZSFAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRSGUZTQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
*Anthony Baggett* || President // Founder
104 N Broadway St.

#2

Tupelo, MS 38804


c. <http://goog_2089110092>662.610.0728 <6626100728>


*BustedSpring.com <http://bustedspring.com/>*- - - - - - - - - - - - - - -
- - - - - - - - - - - - - - - - -

**Comment by Jodyaasanjose** [comment]
On Fri, Feb 9, 2024 at 11:31 AM Anthony Baggett ***@***.***>
wrote:

> Do you have a copy of the old 42 page one? Maybe if I see it in context, I
> can make more adjustments.
>
> On Fri, Feb 9, 2024 at 1:23 PM Jodyaasanjose ***@***.***>
> wrote:
>
> > Yes it went from 52 pages to 46. Before the initial changes it was only
> 42
> >
> > On Fri, Feb 9, 2024 at 8:02 AM Anthony Baggett ***@***.***>
> > wrote:
> >
> > > @ATLCOTech <https://github.com/ATLCOTech> @Jodyaasanjose
> > > <https://github.com/Jodyaasanjose>
> > >
> > > Please check PDF with your feed now. I've tightened up the line
> spacing
> > > some. It should be fewer pages now.
> > >
> > > We are now supporting more languages with PDF, so the font we were
> using
> > > had to be changed to one that supports more characters. Because of the
> > > switch, the layout won't match 100%, but we are hoping to get it as
> > close
> > > as possible.
> > >
> > > Thanks for your patience and your feedback!
> > > Anthony
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8420705>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/BF6R3N37RGZ6VWZFXYRSFYTYSZCANAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRQG4YDK>
>
> >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> > In Grateful Service,
> >
> > MLC
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8422538>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ACF42JIWDY52OAY3AJLWEWDYSZZSFAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRSGUZTQ>
>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
>
> - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
> *Anthony Baggett* || President // Founder
> 104 N Broadway St.
>
> #2
>
> Tupelo, MS 38804
>
>
> c. <http://goog_2089110092>662.610.0728 <6626100728>
>
>
> *BustedSpring.com <http://bustedspring.com/>*- - - - - - - - - - - - - -
> -
> - - - - - - - - - - - - - - - - -
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8422596>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BF6R3N7FM5ZFQKBTWNNYDS3YSZ2SBAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRSGU4TM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
In Grateful Service,

MLC

**Reply by joshreisner** [reply]
hi @Jodyaasanjose i'm guessing you attached some PDFs to the email? those don't come through on Github.

sounds like Anthony just updated the PDF service, if you can try again and let us know if you're good, or come to https://github.com/code4recovery/pdf/discussions/64 and show us before / after that would be good thanks!

**Reply by anchovie91471** [reply]
Thanks @joshreisner! Yes, I believe it should be close enough to the original size and spacing to work now, but @Jodyaasanjose, let me know if that isn't the case!

**Comment by ATLCOTech** [comment]
Thank you so much Anthony!! We appreciate the service and quick
responses/updates.
Sincerely,

Mark J
Tech & Newsletter Chair

If you have a newsletter submission, please send to ***@***.***
.
If you have a website comment, issue or would like to join the tech/web
committee email ***@***.*** ***@***.***>
To contact the steering committee email ***@***.***
Central Office ***@***.***

**Comment by anchovie91471** [comment]
OK, Guys, 

I noticed @ATLCOTech had put an original PDF in this discussion. I used that one to compare and made some more adjustments. 

I think we're there! I got the new one down to the same number of pages as @ATLCOTech had in their original. (60 pages)

**Comment by Jodyaasanjose** [comment]
YES!!!!  It is back to 42 pages

Thanks so much for all you do!

On Fri, Feb 9, 2024 at 12:12 PM Anthony Baggett ***@***.***>
wrote:

> OK, Guys,
>
> I noticed @ATLCOTech <https://github.com/ATLCOTech> had put an original
> PDF in this discussion. I used that one to compare and made some more
> adjustments.
>
> I think we're there! I got the new one down to the same number of pages as
> @ATLCOTech <https://github.com/ATLCOTech> had in their original. (60
> pages)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/pdf/discussions/64#discussioncomment-8422940>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BF6R3NZ4NTTKHWLYNWA3SO3YSZ7LBAVCNFSM6AAAAABC4OBTJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIMRSHE2DA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
In Grateful Service,

MLC
